frontend/src/i18n/locales/de.yaml

```
@@ -154,8 +154,8 @@ video:
   caption: Untertitel
   download:
     title: Download
-    presenter: Kamera
-    slides: Präsentation
+    presenter: Video (Sprecher*in)
```

```suggestion
    presenter: Video (Sprecher:in)
```

Colon is much less disruptive